pyads/ads.py

```diff
@@ -480,6 +483,14 @@ def ams_port(self, port: int) -> None:
             )
         self._adr.port = port
 
+    @property
```

Comment:
Why do you want a public port property?

Reply:
This allows `AdsSymbol` (who references to a `Connection` instance) to call the `adsGetSymbolInfo()` function.
Saying this out loud it might be better to create a new method in `Connection` instead and get rid of this `port` property. Changing it now.